tests/state.tests/blobset_receive.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": 1,
+  "timestamp": "2022-07-13T12:00:10.000Z",
+  "system": {
+    "hardware": {
+      "make": "ACME",
+      "model": "Bird Trap"
+    },
+    "software": {
+      "firmware": "1.0"
+    },
+    "serial_no": "000000",
+    "last_config": "2022-07-11T00:00:10.000Z",
+    "operational": true
+  },
+  "blobset": {
+    "blobs": {
+      "arbitrary_manufacturer_id": {
+        "status": {
+          "message": "Received arbitrary_manufacturer_id",
+          "category": "blobset.blobs.receive",
```

Comment:
for pointset, we ended up with "pointset.point.receive" (singular 'point') -- which is slightly different than "blobset.blobs.receive" -- (plural 'blobs') which is more aligned around the JSON object tree. They're both valid perspectives, but in the face of a strong preference I'd go with the singular version because 1) consistency with pointset, and 2) it makes a bit more sense semantically on the receiving end (as it is scoped to just one blob).